.github/workflows/ci.yml

```diff
@@ -42,12 +42,16 @@ jobs:
         os:
           - "centos-7"
           - "centos-8"
-          - "debian-9"
-          - "ubuntu-1604"
+          - "fedora-latest"
```

Comment:
Is there value in testing fedora as we know it isn't the most stable os?

Reply:
Personally I'd say yes as hashicorp release packages for it and I don't think it's necessarily as much a shifting-sands beast as it's thought of within a version anyway. I'd rather peg to a specific version instead of using latest though so CI builds don't start to fail when a new version is released, but I've had reviews against that in the past so set it to latest here.